src/scheduler/job_info.c

```diff
@@ -3302,13 +3143,8 @@ find_jobs_to_preempt(status *policy, resource_resv *hjob, server_info *sinfo, in
 		schdlog(PBSEVENT_DEBUG2, PBS_EVENTCLASS_JOB, LOG_DEBUG, pjob->name,
 			"Simulation: preempting job");
 
-		po = get_preemption_order(pjob, sinfo);
-		if (po != NULL) {
-			if (po->order[0] == PREEMPT_METHOD_SUSPEND && pjob->job->can_suspend) {
-				pjob->job->resreleased = create_res_released_array(npolicy,pjob);
-				pjob->job->resreq_rel = create_resreq_rel_list(npolicy, pjob);
-			}
-		}
+		pjob->job->resreleased = create_res_released_array(npolicy,pjob);
```

Comment:
You can't always set these.  These are only to be set when we are premepting and when RRTROS is set.  This unfortunately means we'll have to parse out preempt_order from the server and leave the old get_preempt_order code in the scheduler as well :(